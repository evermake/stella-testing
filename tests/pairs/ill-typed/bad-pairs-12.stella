language core;

extend with #pairs;

fn main(x : Nat) -> Nat {
  return {0,0}(0)
}
