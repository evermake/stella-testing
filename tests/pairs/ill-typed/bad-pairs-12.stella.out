TYPE ERROR:
Type Error Tag: [ERROR_NOT_A_FUNCTION]
expected a function type but got
  
{
  Nat, Nat
}

for the expression
  
{
  0, 0
}

in the function call
  
{
  0, 0
}
(0)